src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -3954,6 +3954,48 @@
+
+    ("SveTestTest.template",      new Dictionary<string, string> { ["TestName"] = "SveTestAnyTrue_short_custom1",                                                                          ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "TestAnyTrue",                                                        ["MaskBaseType"] = "Int16",   ["Op1Value"] = "Helpers.CreateAndFillMaskFromFirstElement<Int16>(1)",           ["Op2Value"] = "Helpers.CreateAndFillMaskFromSecondElement<Int16>(1)",                      ["ValidateEntry"] = "result != true"}),
+    ("SveTestTest.template",      new Dictionary<string, string> { ["TestName"] = "SveTestFirstTrue_short_custom1",                                                                        ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "TestFirstTrue",                                                      ["MaskBaseType"] = "Int16",   ["Op1Value"] = "Helpers.CreateAndFillMaskFromLastElement<Int16>(1)",            ["Op2Value"] = "Helpers.CreateAndFillMaskFromSecondToLastElement<Int16>(1)",                ["ValidateEntry"] = "result != true"}),
+    ("SveTestTest.template",      new Dictionary<string, string> { ["TestName"] = "SveTestLastTrue_short_custom1",                                                                         ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "TestLastTrue",                                                       ["MaskBaseType"] = "Int16",   ["Op1Value"] = "Helpers.CreateAndFillMaskFromFirstElement<Int16>(1)",           ["Op2Value"] = "Helpers.CreateAndFillMaskFromSecondElement<Int16>(1)",                      ["ValidateEntry"] = "result != true"}),
+
+    ("SveExtractVectorTest.template",        new Dictionary<string, string> { ["TestName"] = "SveExtractVector_Byte_1",                                                                                ["Isa"] = "Sve",       ["LoadIsa"] = "Sve", ["Method"] = "ExtractVector",                                                     ["RetVectorType"] = "Vector", ["RetBaseType"] = "Byte",    ["Op1VectorType"] = "Vector", ["Op1BaseType"] = "Byte",   ["Op2VectorType"] = "Vector", ["Op2BaseType"] = "Byte",                                                                ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetByte()",          ["NextValueOp2"] = "TestLibrary.Generator.GetByte()",          ["ElementIndex"] = "1",                                                        ["ValidateIterResult"] = "Helpers.ExtractVector(firstOp, secondOp, ElementIndex, i) != result[i]"}),
```

Comment:
you should reuse the template `SveVecImmTernOpFirstArgTest` that we have for `dotproduct` after you fix the `Op2BaseType`.

Reply:
This template was already based on `ExtractVectorTest` which is a really similar API